### ISSUE TITLE
abandoned workloads report logic error

### ISSUE THREAD

**Issue by jessegoodier** [kubecost/cost-analyzer-helm-chart#1730]
**Describe the bug**  
Customer brough this up and we also have an example on internal demo environment (happy to send link):

abandoned workloads have containers with traffic above the threshold see screenshots below.

**To Reproduce**  
1. Go to 'savings>abandoned workloads'
2. Click on detail of a workload that you know has significant traffic
3. See bytes are far above the selected threshold.

**Expected behavior**  
Only containers under the threshold in report

**Screenshots**  

![image](https://user-images.githubusercontent.com/31039225/194324028-ad8c2adc-f881-41ff-bc8e-4caae5c3f9e3.png)
![image](https://user-images.githubusercontent.com/31039225/194324066-fc67a042-5855-4a7c-8c58-b92be965537f.png)

**Collect logs (please complete the following information):**
let me know if you need logs.

**Comment by AjayTripathy** [comment]
Yeah, though I will say this bug predates a lot of our testing workflow we should be writing an integration test on this API.

**Comment by Adam-Stack-PM** [comment]
ok to close?
